pcdsdevices/ccm.py

```diff
@@ -74,7 +74,7 @@ class CCMCalc(FltMvInterface, PseudoPositioner):
     """
 
     energy = Cpt(PseudoSingleInterface, egu='keV', kind='hinted',
-                 limits=(4, 25))
+                 limits=(4, 25), verbose_name='CCM Photon Energy')
```

Comment:
Added a new verbose name attribute here - not sure if this is ok for now, if not, I can just use the doc (although I had some trouble using the `__doc__` because I was getting all the docstring for the class.)

Reply:
I think adding a new name attribute is very appropriate